.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contribution Guidelines
+<img src="https://yoast-mercury.s3.amazonaws.com/uploads/2013/02/Yoast_Logo_Large_RGB.png" alt="Yoast logo" width="250px" />
+
+Before filing a bug report or a feature request, be sure to read the contribution guidelines.
+
+## How to use GitHub
+We use GitHub exclusively for well-documented bugs, feature requests and code contributions.
+Please follow the instructions in the issue and PR templates. 
+
+To receive free support for ACF Content Analysis for Yoast SEO we have the following channels:
+* [Support forums](https://wordpress.org/support/plugin/acf-content-analysis-for-yoast-seo/) on WordPress.org
+
+## Security issues
+Please do not report security issues here. Instead, email them to security at yoast dot com so we can deal with them securely and quickly.
+
+## I have found a bug
+Before opening a new issue, please:
+* update to the newest versions of WordPress, ACF Content Analysis for Yoast SEO, Yoast SEO and Advanced Custom Fields.
+* search for duplicate issues to prevent opening a duplicate issue. If there is already an open existing issue, please comment on that issue.
+* check our [knowledge base](http://kb.yoast.com) for your issue. There are a lot of common errors documented there with possible solutions.
+* check for [plugin and theme conflicts](https://kb.yoast.com/kb/how-to-check-for-plugin-conflicts/). Please report your findings in the issue.
+* check for [JavaScript errors with your browser's console](https://kb.yoast.com/kb/how-to-find-javascript-errors-with-your-browsers-console/). Please report your findings in the issue.
+* add as much information as possible. For example: add screenshots, relevant links, step by step guides etc.
+
+## I have a feature request
+Before opening a new issue, please:
+* search for duplicate issues to prevent opening a duplicate feature request. If there is already an open existing request, please leave a comment there.
+* add as much information as possible. For example: give us a clear explanation of why you think the feature request is something we should consider for the Yoast SEO plugins.
+
+## I want to create a patch
+Community made patches, localizations, bug reports and contributions are very welcome and ACF Content Analysis for Yoast SEO to become the best it can be.
+
+When contributing please ensure you follow the guidelines below so that we can keep on top of things.
+
+#### Submitting an issue you found
+Make sure your problem does not exist as a ticket already by searching through [the existing issues](https://github.com/Yoast/yoast-acf-analysis/issues). If you cannot find anything which resembles your problem, please [create a new issue](https://github.com/Yoast/yoast-acf-analysis/issues/new).
+
+#### Fixing an issue
+
```

Comment:
Is there a reason for this new line? This is the only place where a new line after a heading is added.